pkg/sentry/event/beacon/eth/v1/fork_choice.go

```diff
@@ -0,0 +1,136 @@
+package event
```

Comment:
super nitpick: should this be `debug_fork_choice.go`?

idk if we should follow the beacon api a bit closer or am I coupling this way too closely? eg.

- `pkg/sentry/event/beacon/eth/v1/events_attestation.go` -> `pkg/sentry/event/beacon/eth/v1/events/attestation.go` [`eth/v1/events`](https://ethereum.github.io/beacon-APIs/?urls.primaryName=dev#/Events/eventstream)
- `pkg/sentry/event/beacon/eth/v2/beacon_block.go` -> `pkg/sentry/event/beacon/eth/v2/beacon/block.go` [`eth/v2/beacon/block/{block_id}`](https://ethereum.github.io/beacon-APIs/?urls.primaryName=dev#/Beacon/getBlockV2)
- `pkg/sentry/event/beacon/eth/v1/debug_fork_choice.go` -> `pkg/sentry/event/beacon/eth/v1/debug/fork_choice.go` [`eth/v1/debug/fork_choice`](https://ethereum.github.io/beacon-APIs/?urls.primaryName=dev#/Debug/getDebugForkChoice)
- `pkg/sentry/event/beacon/eth/v1/debug_fork_choice_reorg.go` -> `pkg/sentry/event/beacon/eth/v1/debug/fork_choice_reorg.go` [`eth/v1/debug/fork_choice`](https://ethereum.github.io/beacon-APIs/?urls.primaryName=dev#/Debug/getDebugForkChoice)

`fork_choice_reorg` doesn't really fit tho.

Reply:
Yeah I also don't really know the solution. That `event/beacon/eth` package maybe shouldn't be versioned? It's the "xatu" implementation of our events, which happen to contain eth structs. They were aligned 1:1 before, but now `fork_choice_reorg` is breaking that. 

I might rename to `debug_` and we can leave it for now. I expect we'll have a clearer idea when we go to add more events that aren't 1:1 with beacon structs, and shuffling around these files won't break anything. You all g with that @Savid?